inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -284,6 +284,7 @@ def extract_authors_from_xml(xml_content):
     undefined_or_empty_inspireid_value_regex = re.compile("undefined|inspire-\s*$", re.IGNORECASE)
     undefined_value_regex = re.compile("undefined", re.IGNORECASE)
     ror_path_value_regex = re.compile("https://ror.org/*")
+    remove_new_line_regex = re.compile(u"\s*\n\s+")
```

Comment:
are we sure that there's always `\s` char after new line?

Reply:
Hmm, at least in the problem record yes, but good point. I could change it also to ```\s*\n\s*```

Reply:
I think it would be safer